### PR TITLE
Better documentation for Contributing

### PR DESCRIPTION
**Pull request by guolinke**

### CODE REVIEW

**Comment by guolinke** [comment]
@StrikerRUS @jameslamb @Laurae2 
I think we should have a page for contributors too. 
like https://github.com/dmlc/xgboost/blob/master/CONTRIBUTORS.md

BTW, I plan to create a [Roadma](https://github.com/microsoft/LightGBM/projects/1) to make the development more smoothly.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+LightGBM has been developed and used by many active community members. Your help is very valuable to make it better for everyone.
+
+- Check for [Roadmap](https://github.com/microsoft/LightGBM/projects/1) and look for [issues with tag "help wanted"](https://github.com/microsoft/LightGBM/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22) and submit pull requests to address them. If you need development guideline, you can check the [Development-Guide](https://github.com/microsoft/LightGBM/blob/master/docs/Development-Guide.rst) or directly ask us in Issues/Pull Requests.
```
Comment (StrikerRUS) [inline CONTRIBUTING.md]: As we have adopted the strategy of closing feature requests and practically have no open issues with `help wanted` tag, I don't think that it makes sense to ask users to look for them anymore.
```suggestion
- Check for the [Roadmap](https://github.com/microsoft/LightGBM/projects/1) and the [Feature Requests Hub](https://github.com/microsoft/LightGBM/issues/2302), and submit pull requests to address chosen issue. If you need development guideline, you can check the [Development Guide](https://github.com/microsoft/LightGBM/blob/master/docs/Development-Guide.rst) or directly ask us in Issues/Pull Requests.
```

**Comment by guolinke** [comment]
Thanks @StrikerRUS , we may need another re-organization round for feature request page: https://github.com/microsoft/LightGBM/issues/2302

And could you help to create a Contributors page?

**Comment by StrikerRUS** [comment]
@guolinke 
> And could you help to create a Contributors page?

To be honest, I don't see any point of creating list of all contributors. GitHub already tracks all contributors for us: https://github.com/microsoft/LightGBM/graphs/contributors. For "Project Management Committee(PMC)" we already have a list of core team members in our FAQ: https://github.com/microsoft/LightGBM/blob/master/docs/FAQ.rst#critical-issues.

**Comment by guolinke** [comment]
Okay, I forgot that we have that page.

**Comment by jameslamb** [comment]
I agree with @StrikerRUS that we don't need to explicitly list out all contributors, since we get that from GitHub already.

I am excited  for the Roadmap idea, makes sense.

File: README.md [hunk 2]
```diff
@@ -94,14 +94,8 @@ Support
...
 
-LightGBM has been developed and used by many active community members. Your help is very valuable to make it better for everyone.
-
-- Contribute to the [tests](https://github.com/microsoft/LightGBM/tree/master/tests) to make it more reliable.
-- Contribute to the [documentation](https://github.com/microsoft/LightGBM/tree/master/docs) to make it clearer for everyone.
-- Contribute to the [examples](https://github.com/microsoft/LightGBM/tree/master/examples) to share your experience with other users.
-- Look for [issues with tag "help wanted"](https://github.com/microsoft/LightGBM/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22) and submit pull requests to address them.
-- Add your stories and experience to [Awesome LightGBM](https://github.com/microsoft/LightGBM/blob/master/examples/README.md). If LightGBM helped you in a machine learning competition or some research application, we want to hear about it!
-- [Open an issue](https://github.com/microsoft/LightGBM/issues) to report problems or recommend new features.
+Check [Cortributing](https://github.com/microsoft/LightGBM/blob/master/CONTRIBUTING.md) page.
```
Comment (jameslamb) [inline README.md]:
```suggestion
Check [CONTRIBUTING](https://github.com/microsoft/LightGBM/blob/master/CONTRIBUTING.md) page.
```

**Pull request merged**